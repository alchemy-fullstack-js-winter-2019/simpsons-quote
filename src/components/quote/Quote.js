import React from 'react';
import PropTypes from 'prop-types';

function Quote({ quote, name, image }) {
  return (
    <>
    <p>{quote}</p>
    <p>{name}</p>
    <p><img src={image} /></p>
    </>
  );
}

Quote.propTypes = {
  quote: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};
export default Quote;
