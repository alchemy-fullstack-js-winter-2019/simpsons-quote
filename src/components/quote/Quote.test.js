import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('Quote', () => {
  it('matches a snapshot', () => {
    const quote = 'quote';
    const name = 'name';
    const image = 'image';

    const tree = renderer.create(
      <Quote fetch={jest.fn()} quote={quote} name={name} image={image} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
