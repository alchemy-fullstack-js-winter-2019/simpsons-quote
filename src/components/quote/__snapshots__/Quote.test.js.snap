// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote matches a snapshot 1`] = `
Array [
  <p>
    quote
  </p>,
  <p>
    name
  </p>,
  <p>
    <img
      src="image"
    />
  </p>,
]
`;
